wallet-connect/dapp.md

```diff
@@ -3,6 +3,8 @@
 Trust uses a forked version of WalletConnect with aditional methods to allow **dApp** developers
 to sign transactions [for any blockchain](https://github.com/TrustWallet/wallet-core/blob/master/docs/coins.md). 
 
+> We're currently supporting **Ethereum**, **Binance Coin** and **Cosmos**. More chains will be added soon.
```

Comment:
can you use our standard supported blockchain block: `https://github.com/TrustWallet/blockatlas#block-atlas-by-trust-wallet`, same used for WalletCore